radio/apps/schedules/models.py

```diff
@@ -262,10 +251,9 @@ def between(cls, after, before, exclude=None, live=False, schedule_board=None):
 
     @classmethod
     def schedule(cls, dt, exclude=None):
-        list_schedules = cls.objects.filter(
-            programme__start_date__lte=dt, programme__end_date__isnull=True
-        ) | cls.objects.filter(
-            programme__start_date__lte=dt, programme__end_date__gte=dt
+        list_schedules = (
+            cls.objects.filter(programme__start_date__lte=dt, programme__end_date__isnull=True) |
+            cls.objects.filter(programme__start_date__lte=dt, programme__end_date__gte=dt)
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.